src/cascadia/TerminalApp/AppCommandlineArgs.cpp

```diff
@@ -187,6 +187,10 @@ void AppCommandlineArgs::_buildParser()
                     _windowTarget,
                     RS_A(L"CmdWindowTargetArgDesc"));
 
+    _app.add_option("-s,--saved",

```

Comment:
idle thought: what if someone's already got a layout open with a bunch of windows, then does `wt -w foo -s 0`? Do we tell window `foo` to open the 0th saved layout? Or will it only do that on actual startup, and not for a commandline dispatched to it?

Reply:
Right now it'll just create a new tab since the logic to load the saved layout is in `OnFirstLayout`.

Reply:
nice okay, thanks for confirming 👍